src/70_User_Consent/Advanced_User_Consent_Flow.html

```diff
@@ -113,27 +117,32 @@
                   <li>{{.}}</li>
                 </template>
               </amp-list>
-              <button on="tap:myConsent.accept" class="ampstart-btn ampstart-btn-secondary caps m1">Accept</button>
+              <button on="tap:myConsent.accept" class="ampstart-btn ampstart-btn-secondary caps m1">Confirm</button>
             </div>
             <div id="choice2" hidden class="message">
               <p class="m1">This is some more information about this choice.</p>
-              <button on="tap:myConsent.reject" class="ampstart-btn ampstart-btn-secondary caps m1">Reject</button>
+              <button on="tap:myConsent.reject" class="ampstart-btn ampstart-btn-secondary caps m1">Confirm</button>
```

Comment:
Is it correct to show Confirm here as well?

Reply:
It felt right for me (Rudy wanted the 'reject' one to be 'confirm', and having 'accept' and 'confirm' seemed a bit weird). It feels more consistent like that. But we can check back if you prefer.